app/javascript/Greetings.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from "react";
+
+const Greetings = () => {
+  const [greetings, setGreetings] = useState([]);
+  const [randomGreeting, setRandomGreeting] = useState("");
+
+  useEffect(() => {
+    fetch("/greetings.json")
+      .then((response) => response.json())
+      .then((data) => {
+        setGreetings(data);
+        const randomIndex = Math.floor(Math.random() * data.length);
+        setRandomGreeting(data[randomIndex]?.body || "");
+      })
+      .catch((error) => console.error("Error fetching greetings:", error));
+  }, []);
```

Comment:
- [ ] Fantastic progress on your implementation! 🚀 Your dedication to this project is truly commendable. As we delve into the code, I've got a couple of pointers that will not only enhance the quality of your work but also align it more closely with the project requirements:

   - **Consider integrating `redux` for data state management**: The project requirements specifically mention the creation of a `store`, an `action`, and a `reducer` to connect with your `API` endpoint for retrieving random messages. I've attached a screenshot of the relevant requirements below:

   <img width="659" alt="Screenshot 2023-12-13 at 04 01 55" src="https://github.com/prg-04/React_Rails_Hello_World/assets/107192076/12b59b77-3e7b-4ccd-9a3f-e74daf5d1dd6">

These additions will not only meet the specifications but also streamline your data management, providing a more robust foundation for your project.

Happy coding! 🚀👨‍💻😊